doc/api/debugger.md

```diff
@@ -202,3 +202,35 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+Tracing can be configured by additionally specifying a configuration file using the
+`--trace-config` flag:
+
+```txt
+node --enable-trace --trace-config=trace-config.json server.js
+```
+
+where `trace-config.json` may override any of the default configuration values:
+
+```javascript
+{
+  "record_mode": "record-continuously",
+  "enable_sampling": true,
+  "enable_systrace": true,
+  "enable_argument_filter": true,
+  "included_categories": [ "v8", "node" ],
+  "excluded_categories": [ ]
```

Comment:
This is used by Chrome since they specify enabled categories using regular expressions which can be refined by excluded categories. The node/v8 category parser doesn't support regexes so this isn't needed at this time.